### DISCUSSION TITLE
Mass of the Milky Way revisisted

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#156] [Astrophysics and Physics]
Expanding universe proponents often cite the size of the Milky Way relative to more distant ones. Well...

https://phys.org/news/2023-09-revisited-mass-milky-smaller-cosmology.html

>  The total mass of the Milky Way is estimated to be only two hundred billion times that of the sun (2.06 x 1011 solar masses), marking a significant downward revision—approximately four to five times lower than previous estimates.

**Comment by bgreenman** [comment]
The result shows that the cold dark matter concept isn't as viable as mainstream theorists thought

**Comment by RedshiftDrift** [comment]
The *mass of the Milky Way* is not as important as the *rotation curve of the Milky Way* reported by the paper.  A Keplerian rotation curve means, for some astronomers, no "dark matter".

**Reply by bgreenman** [reply]
Correct idea

**Reply by budrap00** [reply]
It's hard to know what to make of this given it's just a press release. For a nuanced discussion of the Gaia data see McGaugh's [recent treatment](https://tritonstation.com/2023/09/21/recent-developments-concerning-the-gravitational-potential-of-the-milky-way-iii-a-closer-look-at-the-rar-model/comment-page-1/#comment-21756) which doesn't seem to find any significant mass discrepancy nor a Keplerian decline.

**Reply by RedshiftDrift** [reply]
> it's just a press release

The original paper [is on arXiv](https://arxiv.org/abs/2309.00048)

**Comment by bgreenman** [comment]
Problem is we are still trying to figure out the level of rotation curves and degree of mass discrepancies
